server/plugin.go

```diff
@@ -147,121 +140,73 @@ func (p *Plugin) getOAuthConfig() (*oauth2.Config, error) {
 	}, nil
 }
 
-type ZoomUserInfo struct {
-	ZoomEmail string
-
-	// Zoom OAuth Token, ttl 15 years
-	OAuthToken *oauth2.Token
-
-	// Mattermost userID
-	UserID string
-
-	// Zoom userID
-	ZoomID string
-}
-
-type AuthError struct {
-	Message string `json:"message"`
-	Err     error  `json:"err"`
-}
-
-func (ae *AuthError) Error() string {
-	errorString, _ := json.Marshal(ae)
-	return string(errorString)
-}
-
-func (p *Plugin) storeZoomUserInfo(info *ZoomUserInfo) error {
+func (p *Plugin) storeOAuthInfo(info *zoom.OAuthInfo) error {
```

Comment:
@larkox Good point. The reason I went with `OAuthInfo` was that it's only utilized in the OAuth flow and I wanted to emphasize that. Also, since this type is part of the `zoom` package, it feels redundant to prefix it with `Zoom` but it's just a personal opinion 🙂 

What are your thoughts about calling it `OAuthUserInfo`?

Reply:
I like OAuthUserInfo